apimodels/agent_models.go

```diff
@@ -102,6 +102,9 @@ type EndTaskResponse struct {
 }
 
 type CreateHost struct {
+	//option to parse params from a file
+	File string `yaml:"file"`
```

Comment:
It feels a little meta to put File in CreateHost since it defines the parameters _for_ the struct. I would argue for putting it into the larger command createHost (sorry these all have the same names) and directly assigning it as `c.File = params["file"]`